modules/images/dominant-color/load.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Module Name: Dominant Color
+ * Description: Adds support to store dominant-color for an image and create a placeholder background with that color.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Add the dominant color metadata to the attachment.
+ *
+ * @param array $metadata The attachment metadata.
```

Comment:
Every new function need to have `@since n.e.x.t` tag. Could you please add it to all new functions and methods?
```suggestion
/**
 * Add the dominant color metadata to the attachment.
 *
 * @since n.e.x.t
 *
 * @param array $metadata The attachment metadata.
```

Reply:
@pbearne This hasn't been addressed yet, there's still no `@since` annotation here.